### PR TITLE
add new userscript to run local command and update README.md

### PR DESCRIPTION
**Pull request by towalker**
The run_local_cmd userscript is mainly for convenience, to be able to run commands without leaving qutebrowser.

### CODE REVIEW

File: misc/userscripts/run_local_cmd [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+   by Tiago Oliveira Weber (towalker), 2020
+"""
+
+import os, sys
+import subprocess
+
+max_message_lines = 10
+max_message_size = 20
+
+w = os.pipe()
```
Comment (rien333) [inline misc/userscripts/run_local_cmd:31]: Is it correct that this remains unused?

**Comment by rien333** [comment]
The main difference between this and `:spawn` is that this userscript redirects the command's stdout to qutebrowser's message area, correct? If so, wouldn't it be a good idea to do the same for stderr (if this not happens already)?

**Comment by The-Compiler** [comment]
The `:spawn` command also takes a `--output-messages` (`-m`) flag since qutebrowser v1.9.0 - so I don't think there's any need for this userscript (other than the message count/length limit, perhaps), is there?

File: misc/userscripts/run_local_cmd (hunk 1 above)
Comment (towalker) [reply misc/userscripts/run_local_cmd:31]: Indeed it can be deleted.

**Comment by towalker** [comment]
I was not aware of the spawn output messages. Therefore, I agree there is no need for the userscript.

**Comment by The-Compiler** [comment]
Alright, closing this then - thanks for the contribution anyways! :+1:

**Pull request closed without merging**